MetaMask/metamask-extension
# Increase token rates controller interval

## **Description**

This reduces the request load we put on coingecko

## **Related issues**

Fixes: #

## **Manual testing steps**

1. Open metamask
2. Import tokens
3. See request to coingecko
4. Leave background console open and wait 65 minutes, there should be 2 and only 2 more requests to coingecko

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
